devicetypes/smartthings/smartsense-button.src/smartsense-button.groovy

```diff
@@ -273,6 +273,8 @@ def configure() {
     // Device-Watch allows 2 check-in misses from device + ping (plus 1 min lag time)
     // enrolls with default periodic reporting until newer 5 min interval is confirmed
     sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+    // Sets up low battery threshold reporting
+    sendEvent(name: "DeviceWatch-Enroll", value: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, scheme: "TRACKED", checkInterval: 60 * 12, lowBatteryThresholds: [15, 7, 3]])
```

Comment:
@aarontimbo do we need to include `offlinePingable: "1"` here as well or will the settings set via the `checkInterval` event still be used?

Reply:
@greens should add `displayed: false`